### ISSUE TITLE
[Bug]: Can't progress - Mekanism Infused Alloy Recipe missing since last update

### ISSUE THREAD

**Issue by StevenSeegal** [AllTheMods/All-the-mods-9-Sky#636]
### Possible Fixes

Yes

### Modpack Version

1.1.0

### What happened?

I updated the pack today to version and was surprised Mekanism Infused Alloy Recipe is missing. (Iron in Metallurgic Infuser with Redstone). JEI doesn't show a Metallurgic Infuser recipe anymore, only 5-6 via a Quest Reward.
It only affect that one, the Diamond just shows up in JEI (Both as Quest reward and Metallurgic Infuser Recipe) and can be crafted.

**Without this  Infused Alloy I can't craft a lot of Recipes anymore. Pipes, Energy pipes, etc. and it''s also used for certain other crafting recipes.**

![Infused_Alloy_1](https://github.com/user-attachments/assets/158b59de-0af5-4237-b666-8c812b1eabdb)
![Infused_Alloy_2](https://github.com/user-attachments/assets/f1714a20-915c-40d2-9a6a-fef1cc34fd6d)
![Diamond_Alloy](https://github.com/user-attachments/assets/234b3b4d-47ad-4204-a323-c99211a35aff)

### latest.log

https://gist.github.com/StevenSeegal/59b28af826f71c99f52ec8c3bbc93bf2

**Comment by StevenSeegal** [comment]
I reverted back to version 1.0.9 for now so I can continue playing. It's working in 1.0.9 but missing in 1.1.0
Here's a screenshot of the Infused Alloy on 1.0.9:
![Infused_Alloy_1 0 9](https://github.com/user-attachments/assets/9e4d872d-b782-4fc3-a5a1-62fba7e7ffce)

**Comment by ReikoAldrakar** [comment]
Can you look at #610? I had the problem before too but just restarting the game helped me

**Comment by StevenSeegal** [comment]
> Can you look at #610? I had the problem before too but just restarting the game helped me

Before I created this issue I've restarted the game multiple times and even rebooted my PC, but that didn't fix the issue.
Not to brag but I'm very familiar with Modded MineCraft. I've written about 10+ Mods years ago for a ModPack for a mate and 1 is still on CurseForge so I already infested about 1-2 hours trying to solve this missing recipe issue I had after updating the pack from 1.0.9 to 1.1.0.

However I'm happy to report reading though[ #610](https://github.com/AllTheMods/All-the-mods-9-Sky/issues/610) and the "clear your config" hint did do the trick. Thanks for pointing me towards a fix for me.
